.github/workflows/docker-hub.yml

```diff
@@ -1,4 +1,5 @@
 name: Docker Hub Workflow
+run-name: Docker Hub Workflow
```

Comment:
Any particular reason to use `run-name` over `name` ? (I don't think this is a blocker for merging the PR but if there is a practical reason it might be useful to document it…)